package.json

```diff
@@ -7,6 +7,8 @@
     "node": "^12.18"
   },
   "scripts": {
+    "axe:ci": "cross-env URLS=\"http://localhost:3000, http://localhost:3000/activity-reports, http://localhost:3000/admin\" yarn axe:sub",
+    "axe:sub": "cross-env-shell \"axe $URLS --dir ./reports/\"",
```

Comment:
@jasalisbury It's hard to find solutions that will work for windows, linux, and macOS.  Do you think this is better or worse?

Reply:
Where is `$URLS` defined?

Reply:
oh I think I get it now... I'd probably rather the first solution

Reply:
Agreed. Changing back.